src/harness/testcases/testdata/ppa_record_0.json

```diff
@@ -4,6 +4,10 @@
   "usage": "PPA",
   "terminated": false,
   "ppaInfo": {
+    "palId": ["test_fcc_id_pal_1"],
```

Comment:
These four fields are filled in the make...Consistent() function, so there's no need to add them here. We should add the region to ppa_record_3.json but otherwise these changes can be reverted.

Reply:
Slight modification: I agree that adding an *empty list* for the CBSD reference IDs makes sense. Thank you!

Reply:
do we want to make it empty or with a single fake id? I think it make sense to have at least on cbad id as otherwise the PPA is not valid.

Reply:
A fake ID would confuse the SAS. Empty is better, and is allowed in our definition of the testing API.

Reply:
changed cbsdReferenceId to empty list